app/models/user.rb

```diff
@@ -9,5 +10,6 @@ class User < ApplicationRecord
 
   validates :first_name, presence: true
   validates :last_name, presence: true
+  validates :avatar_photo, presence: true
```

Comment:
Ideally a profile picture should be optional, so perhaps we should not enforce presence.

Reply:
Agree with both, I merge it for now but let's keep it in mind for the next project.